io/subsystem/src/main/java/org/wildfly/extension/io/WorkerResourceDefinition.java

```diff
@@ -77,7 +77,7 @@
             .build();
     static final OptionAttributeDefinition WORKER_TASK_KEEPALIVE = new OptionAttributeDefinition.Builder(Constants.WORKER_TASK_KEEPALIVE, Options.WORKER_TASK_KEEPALIVE)
             .setFlags(AttributeAccess.Flag.RESTART_ALL_SERVICES)
-            .setDefaultValue(new ModelNode(60))
+            .setDefaultValue(new ModelNode(60_000))
```

Comment:
This needs to be fixed as part of this PR.

It's not tricky, it's a standard change in a default value. If the user submits undefined we send 60000 to the slave, not undefined. So the legacy servers behave in a manner consistent with the current ones.